### DISCUSSION TITLE
Is there a way to replace nerd font with another font/icons set?

### DISCUSSION THREAD

**Discussion by dmshvetsov** [LazyVim/LazyVim#2471] [Q&A] [answered]
Is there a way to replace nerd font with another font/icons set?

**Comment by dpetka2001** [comment]
If you use your own custom icons and you can find a font that supports them, then I don't see any reason why you can't do it. But Lazyvim uses icons that need a Nerd font to be rendered. So, there's that.

**Reply by dpetka2001** [reply]
I don't know which terminal emulator you're using, but if you use `kitty` you can use `symbol_map` to define a specific font to render the icons and then use another font of whatever family you want for rendering text. Even different families for normal, italics and bold. I'm not familiar with other terminal emulators, but maybe there's a way to do something similar.

**Reply by dmshvetsov** [reply]
that's an option, thanks for sharing

**Reply by dpetka2001** [reply, marked as answer]
You can see here in my [dotfiles](https://github.com/dpetka2001/dotfiles/blob/7b9c05a503b62c280d2fdae07e3c1b1b13670193/dot_config/kitty/kitty.conf#L4-L13) how I do it for `kitty`. I use different fonts for normal and italics and only specify Nerd fonts for `symbol_map`. I'm not familiar with other terminal emulators, but there might be some that allow you to do the same. You would have to read the docs of the terminal emulator you use to see if you can do something similar.

**Reply by dmshvetsov** [reply]
Got it, so it is depends on the terminal you use and Wezterm that I use there is no such option as `symbol_map`. And btw NerdFont is bundled in Wezterm.

**Reply by dpetka2001** [reply]
Not familiar with Wezterm, so I don't really know. Maybe they have another option that does the same thing or another way of defining different fonts. You would have to read their docs.

**Comment by dmshvetsov** [comment]
> If you use your own custom icons and you can find a font that supports them, then I don't see any reason why you can't do it. But Lazyvim uses icons that need a Nerd font to be rendered. So, there's that.

are you saying that Nerd font is "hardcoded" into lazyvim?

**Reply by dpetka2001** [reply]
No, i'm saying that [nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons), which LazyVim uses, has as a requirement for Nerd Fonts. You can see that in their README.
